### PR TITLE
Add Worms imagery

### PR DESCRIPTION
**Pull request by coolultra1**
https://wiki.openstreetmap.org/wiki/DE:Permissions/Luftbild_Worms

Not a lot to say, plz merge

### CODE REVIEW

**Comment by cicku** [comment]
Please remove imageries prior to 2012 unless you really have a good reason.

**Comment by coolultra1** [comment]
There is no reason not to include the old imagery. Not only for OHM, it can
be used to see features long gone or obstructed by trees in the latest
aerial imagery.

**Comment by cicku** [comment]
I don't think so. Long gone features should not be added

**Comment by coolultra1** [comment]
I still think, there is no reason not to add it.

**Comment by coolultra1** [comment]
> Please remove imageries prior to 2012 unless you really have a good reason.

Actually, you should be the one giving me the reason to remove. This project is meant to be a complete index, not just what might be needed.

**Comment by rbuffat** [comment]
@coolultra1 The topic of how long imagery should be included in ELI is a bit controversial. See https://github.com/osmlab/editor-layer-index/issues/1446 for an ongoing discussion. 

As you probably know these imageries the best, what do you think how useful imagery older than 2012 will be for mappers mapping features that currently exist on the ground? I guess the newest imagery is the most useful, for older imagery there might be certain features that are better visible. Do you think there will be features that are not visible in either the imagery of 2020, 2016 or 2012 that still exist today?

**Comment by cicku** [comment]
You are right, we are still discussing it. https://github.com/osmlab/editor-layer-index/issues/1446.

```
 create mode 100644 sources/europe/de/Worms-2003.geojson
 create mode 100644 sources/europe/de/Worms-2008.geojson
 create mode 100644 sources/europe/de/Worms-2012.geojson
 create mode 100644 sources/europe/de/Worms-2016.geojson
 create mode 100644 sources/europe/de/Worms-2020.geojson
```

Something from 10 years ago could be/still be useful, but 20 years? I don't think so. Because you have something from 2 years ago submitted, it is hard to make me believe that newer is not better. It is enough to compare between 2012, 2016 and 2020.

**Comment by coolultra1** [comment]
So? @cicku @rbuffat? Is this mergable this way?

**Comment by coolultra1** [comment]
@cicku, @rbuffat, this has unfortunately seemed to have caught up in discussion, but that shouldn't be a reason to delay merging this. People are still using old imagery around here, please merge this!

And if its really, really necessary, delete the imagery, my god, I don't care.

**Comment by andrewharvey** [comment]
In my view we should accept this until there is a decision around #1446 and subsequent project policy.

File: sources/europe/de/Worms-2003.geojson [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "type": "Feature",
+    "properties": {
+        "name": "Worms 2003",
+        "type": "wms",
+        "url": "https://geoportal-worms.de/ogc/wms/luftbild-2003?LAYERS=D763F3C28EC14D8BAB4C307D33306FAF&STYLES=&FORMAT=image/png&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
+        "license_url": "https://wiki.openstreetmap.org/wiki/DE:Permissions/Luftbild_Worms",
```
Comment (andrewharvey) [inline sources/europe/de/Worms-2003.geojson:7]: I'm not really in a position to assess the legalities here, but this sounds like a clear okay that they allow derived data like traced features to be owned by you, just we can't publish the actual imagery.

**Review by andrewharvey — APPROVED** [review]
This change looks good to me, I'm happy to accept it. @rbuffat did you have any opinion?

File: sources/europe/de/Worms-2003.geojson (hunk 1 above)
Comment (coolultra1) [reply sources/europe/de/Worms-2003.geojson:7]: @andrewharvey Yeah, I wrote the city myself and, as described on the Wiki page, they clearly stated, derived features may be used.

**Comment by coolultra1** [comment]
> This change looks good to me, I'm happy to accept it. @rbuffat did you have any opinion?

Seems @rbuffat has got no opinion on this? Please, I'm begging anyone to merge this.

**Pull request merged**